### PR TITLE
Run scratch packages in parallel

### PR DESCRIPTION
**Pull request by pcreech**
This isn't necessarily the final implementation, but opening the PR for conversation.

I ran the entire nodejs group in foreman-packaging with these modifications and ran into no issues from what I could tell.

This will allow us to send all the packages over to koji in a big lump, and let koji work through the queue instead of having to do one package at a time.

### CODE REVIEW

**Comment by zjhuntin** [comment]
I'd prefer if we had some sort of option to run them in parallel, like [test rpm parallel] or something I think defaulting to it is more than we want.

**Comment by ehelms** [comment]
If dropping the 1 limit actually works now, I'd be in favor of it. We could increase the serial to some greater time at first or a percentage. The real question is do we want Obal to limit the throughput here or the build system itself.

**Comment by ekohl** [comment]
The upside of letting obal do it is that cancelling a huge PR still has some effect. If everything is unleashed on a 100 packages but then another PR is pushed for a typo fix the build system will have a huge queue.

Perhaps we should start with allowing 10 concurrent builds as a fair compromise. It should cover most PRs

File: obal/data/playbooks/scratch/scratch.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 - hosts:
     - packages
-  serial: 1
+  # serial: 1
```
[the remark this answers is not in the document]
Comment (pcreech) [reply obal/data/playbooks/scratch/scratch.yaml]: Neither am I :D  this wasn't meant as final implementation, just a conversation starter

File: obal/data/ansible.cfg [hunk 2]
```diff
@@ -9,3 +9,4 @@ display_skipped_hosts = false
 roles_path = ./roles/
 library = ./modules
 module_utils = ./module_utils
+strategy = free
```
[the remark this answers is not in the document]
Comment (pcreech) [reply obal/data/ansible.cfg]: We can probably remove strategy all together, while modifying the serial variable

**Comment by pcreech** [comment]
Setting this wide open was more of a stress test to see if it could.  Agree it's not the most expedient play.

Batching up a significant number (10 is probably a good starting point) should be enough to at least provide a good sample size to see if it is helpful

**Comment by evgeni** [comment]
:+1: on doing batches of 10 and not modifying `serial` :)

**Comment by ehelms** [comment]
@evgeni did you mean `strategy` or `serial` ?

**Comment by evgeni** [comment]
Strategy!

**Comment by pcreech** [comment]
Looks like this is merging on the consensus of modifying serial for a given playbook (i.e. scratch).

I'll write up a trello card with the details so we can act upon it at an appropriate time.

:+1: or :-1: this post if you agree or disagree.  I'll close this PR out after a day(ish) has past

**Comment by ekohl** [comment]
:+1: to modifying the serial for a given playbook.

Disclaimer: I didn't actually test this and it may have other nasty side effects.

**Pull request closed without merging**